README.md

```diff
@@ -300,16 +313,16 @@ Tested builds are built on: `go version go1.11.4 linux/amd64`
 
 Warning: may be not up-to-date feature list.
 
-The [`docopts.sh`](docopts.sh) is an extra bash library that you can source in your CLI script.
+The [`docopts.sh`](docopts.sh) helper is an extra bash library that you can source in your shell script.
```

Comment:
Same as above, we can now move `doctops.sh` references to a new dedicated  section.

Reply:
OK - I am trying to do a little cleanup.  Larger doc changes could be done separately - perhaps moving to a `docs` directory with several files that supplement this README?